collect_app/src/test/java/org/odk/collect/android/activities/FirstLaunchActivityTest.kt

```diff
@@ -25,13 +25,17 @@ import org.odk.collect.android.projects.QrCodeProjectCreatorDialog
 import org.odk.collect.android.support.CollectHelpers
 import org.odk.collect.android.version.VersionInformation
 import org.odk.collect.strings.localization.getLocalizedString
+import org.odk.collect.testshared.ActivityScenarioLauncherRule
 
 @RunWith(AndroidJUnit4::class)
 class FirstLaunchActivityTest {
 
+    @get:Rule
+    val launcherRule = ActivityScenarioLauncherRule()
```

Comment:
Why here you need this custom rule but in the previous commit you could use `ActivityScenarioRule`. Can't you do the same here?

Reply:
It's needed for the last test: `ActivityScenarioRule` launches the Activity before the test but in `The app name with its version should be displayed` we want to launch the Activity after overriding some dependencies. This kind of situation, and when we launch multiple Activity objects, are what we require `ActivityScenarioLauncherRule` for. Personally, I think it's how `ActivityScenarioRule` should behave (I'm going to be giving this feedback to AndroidX Test).